proxy/src/coco/peer.rs

```diff
@@ -10,15 +10,27 @@ use librad::meta::project;
 use librad::meta::user;
 use librad::net::discovery;
 pub use librad::net::peer::{PeerApi, PeerConfig};
-use librad::uri::RadUrn;
+use librad::uri::{Path, Protocol, RadUrn};
 use radicle_surf::vcs::git::{self, git2};
 
+use super::source;
 use crate::error;
 use crate::project::Project;
 
 /// Export a verified [`user::User`] type.
 pub type User = user::User<entity::Verified>;
 
+/// A bundled type for the revisions API response containing information about
+/// the user and its branches and tags scoped by a project.
+pub struct Remote {
+    /// The user identity.
+    pub user: user::User<entity::Draft>,
```

Comment:
TODO: Should probably be `User<Verified>` i.e. `User` from line 21.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/636/commits/b2536a4d58f78b8368eb9c521e5efe8932eb2787